packages/server/src/common/command/command.ts

```diff
@@ -108,12 +110,14 @@ export class CompoundCommand implements Command {
         const alreadyRedone: Command[] = [];
 
         try {
-            this.commands.forEach(command => {
+            for (const command of this.commands) {
```

Comment:
Should we not also await the `redo` in the line below to make sure that the order is properly kept and there are no conflicts?